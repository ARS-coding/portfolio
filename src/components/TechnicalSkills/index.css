.skillDiv {
  text-align: center;
  width: 120px;
}

.skillIcon {
  font-size: 5rem;
}

.svgIcon {
  height: 80px;
}

.bootstrapLogo {
  color: #563d7c;
}

.gitLogo {
  color: #f34f29;
}

.reactLogo {
  color: #61dbfb;
}

.javaScriptLogo {
  color: #f0db4f;
}

.cssLogo {
  color: #2965f1;
}
