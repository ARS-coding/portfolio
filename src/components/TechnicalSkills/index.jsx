import React from 'react';
import { Row } from 'react-bootstrap';
import './index.css';
import reduxLogo from '../../img/reduxLogo.svg';
import firebaseLogo from '../../img/firebaseLogo.svg';
import antDesignLogo from '../../img/antDesignLogo.svg';
import htmlLogo from '../../img/htmlLogo.svg';
import jestLogo from '../../img/jestLogo.svg';
import i18nLogo from '../../img/i18nLogo.svg';

const TechnicalSkills = () => {
  return (
    <>
      <h2 id='technicalSkills' className='homePageHeader'>
        Technical Skills
      </h2>
      <Row className='justify-content-around'>
        <div className='skillDiv'>
          HTML5 <br />
          <img src={htmlLogo} alt='html logo' className='svgIcon' />
        </div>
        <div className='skillDiv'>
          CSS <br />
          <i class='fab fa-css3-alt skillIcon cssLogo'></i>
        </div>
        <div className='skillDiv'>
          JavaScript <br />
          <i class='fab fa-js-square skillIcon javaScriptLogo'></i>
        </div>
        <div className='skillDiv'>
          React <br />
          <i class='fab fa-react skillIcon reactLogo'></i>
        </div>
        <div className='skillDiv'>
          Git <br />
          <i class='fab fa-git skillIcon gitLogo'></i>
        </div>
        <div className='skillDiv'>
          Github <br />
          <i class='fab fa-github skillIcon'></i>
        </div>
        <div className='skillDiv'>
          Bootstrap <br />
          <i class='fab fa-bootstrap skillIcon bootstrapLogo'></i>
        </div>
        <div className='skillDiv'>
          Ant Design <br />
          <img src={antDesignLogo} alt='ant design logo' className='svgIcon' />
        </div>
        <div className='skillDiv'>
          Firebase <br />
          <img src={firebaseLogo} alt='firebase logo' className='svgIcon' />
        </div>
        <div className='skillDiv'>
          Redux <br />
          <img src={reduxLogo} alt='redux logo' className='svgIcon' />
        </div>
        <div className='skillDiv'>
          i18next <br />
          <img src={i18nLogo} alt='i18n logo' className='svgIcon' />
        </div>
        <div className='skillDiv'>
          Jest <br />
          <img src={jestLogo} alt='jest logo' className='svgIcon' />
        </div>
      </Row>
    </>
  );
};

export default TechnicalSkills;
