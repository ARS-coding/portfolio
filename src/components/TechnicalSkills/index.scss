.skillDiv {
  text-align: center;
  width: 130px;
}

.skillIcon {
  font-size: 5rem;
}

.svgIcon {
  height: 80px;
}

.bootstrapLogo {
  color: #563d7c;
}

.gitLogo {
  color: #f34f29;
}

.reactLogo {
  color: #61dbfb;
}

.javaScriptLogo {
  color: #f0db4f;
}

.cssLogo {
  color: #2965f1;
}

@media only screen and (max-width: 434px) {
  .skillRow {
    justify-content: center;
  }

  .skillDiv {
    margin-top: 30px;
  }
}
