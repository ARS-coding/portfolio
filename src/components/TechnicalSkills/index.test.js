import React from 'react';
import renderer from 'react-test-renderer';
import TechnicalSkills from '.';

it('TechnicalSkills renders correctly', () => {
  const tree = renderer.create(<TechnicalSkills />).toJSON();
  expect(tree).toMatchSnapshot();
});
