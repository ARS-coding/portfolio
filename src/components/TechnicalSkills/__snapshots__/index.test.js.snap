// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnicalSkills renders correctly 1`] = `
Array [
  <h2
    className="homePageHeader"
    id="technicalSkills"
  >
    Technical Skills
  </h2>,
  <div
    className="skillRow row"
  >
    <div
      className="skillDiv"
    >
      HTML5 
      <br />
      <img
        alt="html logo"
        className="svgIcon"
        src="htmlLogo.svg"
      />
    </div>
    <div
      className="skillDiv"
    >
      CSS 
      <br />
      <i
        class="fab fa-css3-alt skillIcon cssLogo"
      />
    </div>
    <div
      className="skillDiv"
    >
      JavaScript 
      <br />
      <i
        class="fab fa-js-square skillIcon javaScriptLogo"
      />
    </div>
    <div
      className="skillDiv"
    >
      React 
      <br />
      <i
        class="fab fa-react skillIcon reactLogo"
      />
    </div>
    <div
      className="skillDiv"
    >
      Git 
      <br />
      <i
        class="fab fa-git skillIcon gitLogo"
      />
    </div>
    <div
      className="skillDiv"
    >
      Github 
      <br />
      <i
        class="fab fa-github skillIcon"
      />
    </div>
    <div
      className="skillDiv"
    >
      Bootstrap 
      <br />
      <i
        class="fab fa-bootstrap skillIcon bootstrapLogo"
      />
    </div>
    <div
      className="skillDiv"
    >
      Ant Design 
      <br />
      <img
        alt="ant design logo"
        className="svgIcon"
        src="antDesignLogo.svg"
      />
    </div>
    <div
      className="skillDiv"
    >
      Firebase 
      <br />
      <img
        alt="firebase logo"
        className="svgIcon"
        src="firebaseLogo.svg"
      />
    </div>
    <div
      className="skillDiv"
    >
      Redux 
      <br />
      <img
        alt="redux logo"
        className="svgIcon"
        src="reduxLogo.svg"
      />
    </div>
    <div
      className="skillDiv"
    >
      i18next 
      <br />
      <img
        alt="i18n logo"
        className="svgIcon"
        src="i18nLogo.svg"
      />
    </div>
    <div
      className="skillDiv"
    >
      Jest 
      <br />
      <img
        alt="jest logo"
        className="svgIcon"
        src="jestLogo.svg"
      />
    </div>
  </div>,
]
`;
